packages/cli/src/util/config.ts

```diff
@@ -94,11 +95,12 @@ export async function loadFoundryConfig(): Promise<
     let foundryConfig: FoundryConfig;
     try {
       const fileContent = await fsPromises.readFile(configFilePath, "utf-8");
-      foundryConfig = JSON.parse(fileContent);
-    } catch {
+      const extension = extname(configFilePath);
+      foundryConfig = parseConfigFile(fileContent, extension);
```

Comment:
Given we don't use the extension anywhere outside maybe simpler to just move it into `parseConfigFile`